meta/main.yml

```diff
@@ -17,6 +17,44 @@ galaxy_info:
         # - jammy
         - focal
 
+    - name: Nobara
+      versions:
+        - "37"
+        - "36"
+
+    - name: Fedora
+      versions:
+        - "39" # rawhide
```

Comment:
We should only add here what we are adding to tests. In this case EL and Nobara.

Good example of a role that supports more distros:
https://github.com/geerlingguy/ansible-role-docker/blob/master/meta/main.yml#L12

https://github.com/geerlingguy/ansible-role-docker/blob/master/.github/workflows/ci.yml#L44

Reply:
EL is not supported as it dos not have a Firefox package, this is why I did it how I did it. And everything is based on fedora:
```mermaid
flowchart LR
    Fedora --> CentOS
    Fedora --> RHEL
    CentOS --> CentOSStream
    RHEL --> AlmaLinux
    RHEL --> RockyLinux
    RHEL --> OracleLinux
    Fedora --> Nobara
```
That's why i only did tests for fedora, but I see a reason for adding testing for the other distros. (like RHEL not having a Firefox package)
Adding tests for Nobara might be a bit hard as there are no docker images for it.

Reply:
It is ok like this, we just don't need that many old versions.